package.json

```diff
@@ -32,10 +32,12 @@
     "next": "12.0.7",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-hook-form": "7.22.2"
+    "react-hook-form": "7.22.2",
```

Comment:
Use `yarn install` not `npm install` its added a lock file which isn't relevant to this project because we are using `yarn`